src/main/java/ja/workshops/spring/części/konfiguracja/xml/c_mvc/Samochód.java

```diff
@@ -0,0 +1,47 @@
+package ja.workshops.spring.części.konfiguracja.xml.c_mvc;
+
+import ja.workshops.spring.części.konfiguracja.xml.c_mvc.silnik.Silnik;
+
+/**
+ * Encja samochód, której reprezentacje trzymane są w lokalnej bazie danych {@link RepozytoriumSamochodów}.
```

Comment:
1. Lokalna
2. Encja - skoro używamy terminu, to odnieśmy się do braku adnotacji `@Entity`